src/Magento/FunctionalTestingFramework/Config/MftfApplicationConfig.php

```diff
@@ -157,6 +172,16 @@ public function getDebugLevel()
         return $this->debugLevel ?? getenv('MFTF_DEBUG');
     }
 
+    /**
+     * Returns a boolean indicating whether mftf is generating skipped tests.
+     *
+     * @return boolean
+     */
+    public function allowSkipped()
+    {
+        return $this->allowSkipped ?? getenv('ALLOW_SKIPPED');
```

Comment:
Does mftf or codeception support ALLOW_SKIPPED?

Reply:
MFTF, codeception does not have a notion of `skipped` tests at all.

Reply:
Then we introduce this ALLOW_SKIPPED? in .env?

Reply:
Good point, I was pattern matching other configs. I'll add this to the default + docs